web-registry/src/components/templates/ProjectDetails/ProjectDetails.utils.tsx

```diff
@@ -177,10 +190,10 @@ const getPartyFromMetadata = (
     type: metadataRole?.['@type'].includes('regen:Organization') // covers Organization or OrganizationDisplay
       ? 'ORGANIZATION' // to provide default image
       : 'USER',
-    image: metadataRole?.['schema:image'],
+    image: metadataRole?.['schema:image'] || DEFAULT_PROFILE_COMPANY_AVATAR,
```

Comment:
I guess we could use the same logic as above here, if of type 'USER' display `DEFAULT_PROFILE_USER_AVATAR`

Reply:
6ff655746cadac67a61473cd82f391b4d5d1015c